app/src/main/java/com/navapp/navigation/destination/PlaceAutocomplete.java

```diff
@@ -0,0 +1,25 @@
+package com.navapp.navigation.destination;
+
+import com.google.maps.GeoApiContext;
+import com.google.maps.PlaceAutocompleteRequest;
+import com.google.maps.PlacesApi;
+import com.google.maps.errors.ApiException;
+import com.google.maps.model.AutocompletePrediction;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class PlaceAutocomplete {
+    private final GeoApiContext context;
+    private final PlaceAutocompleteRequest.SessionToken sessionToken;
+
+    public PlaceAutocomplete(GeoApiContext context, PlaceAutocompleteRequest.SessionToken sessionToken) {
+        this.context = context;
+        this.sessionToken = sessionToken;
+    }
+
+    List<AutocompletePrediction> predictions(String address) throws InterruptedException, ApiException, IOException {
```

Comment:
Shouldn't it be `public` if it's for the user?

Reply:
Plus maybe change the name into something more understandable

Reply:
> Shouldn't it be `public` if it's for the user?

woopsy